packages/signals/signal_handler.pony

```diff
@@ -10,19 +10,21 @@ use @pony_asio_event_destroy[None](event: AsioEventID)
 actor SignalHandler
   """
   Listen for a specific signal.
+  If the wait parameter is true, the program will not terminate until the SignalHandler's dispose method is called.
```

Comment:
i would also add: "or if the `SignalNotify` returns false, after handling the signal"
as this also disposes the SignalHandler and unsubscribes from asio.

Reply:
I think that "logic" is already documented in SignalNotify.pony:
```
    Called with the the number of times the signal has fired since this was
    last called. Return false to stop listening for the signal.
```

Reply:
True point. I would add it nonetheless, as it is important for users to understand under which circumstances waiting stops. It is more convenient to users to have this information in one place.

Thank you for the PR, it looks very good.